src/resources/utils.py

```diff
@@ -79,3 +83,34 @@ def parse_user_update():
     parser.add_argument('email', type=str, required=False, store_missing=False)
     args = parser.parse_args()
     return args
+
+
+def aes_encrypt(plaintext) -> bytes:
+    app_setting = current_app.config[AppSetting.FLASK_KEY]
+    secret, key = app_setting.fcm_secret_key.split(':')
+    cipher = AES.new(key.encode('UTF-8'), AES.MODE_CTR, counter=lambda: secret.encode('UTF-8'))
+    return cipher.encrypt(plaintext)
+
+
+def aes_decrypt(ciphertext) -> str:
+    app_setting = current_app.config[AppSetting.FLASK_KEY]
+    secret, key = app_setting.fcm_secret_key.split(':')
+    cipher = AES.new(key.encode('UTF-8'), AES.MODE_CTR, counter=lambda: secret.encode('UTF-8'))
+    return cipher.decrypt(ciphertext).decode('UTF-8')
+
+
+def send_fcm_notification(key, device_id):
+    url = 'https://fcm.googleapis.com/fcm/send'
```

Comment:
Don't use it in here, place it on our `AppSetting`. Also, extract this function from here utils to `notification.py`.